### PR TITLE
Missing closing title tag generates validation error

### PR DESCRIPTION
**Pull request by Iula79**
fixing the missing title tag

### CODE REVIEW

File: src/validations/html/slowparse.js [hunk 1]
```diff
@@ -66,6 +71,7 @@ const errorMap = {
   UNCLOSED_TAG: error => ({
     reason: 'unclosed-tag',
     payload: {tag: error.openTag.name},
+
```
Comment (jwang1919) [inline src/validations/html/slowparse.js]: fyi @outoftime will freak out about this

Comment (Iula79) [reply src/validations/html/slowparse.js]: removed

File: src/validations/mergeValidations.js [hunk 2]
```diff
@@ -4,9 +4,12 @@ import flatten from 'lodash-es/flatten';
 import flatMap from 'lodash-es/flatMap';
 import sortBy from 'lodash-es/sortBy';
 import omit from 'lodash-es/omit';
+import uniqWith from 'lodash-es/uniqWith';
+
 
 function filterErrors(errors) {
-  const groupedErrors = groupBy(errors, 'reason');
+  const dedupedErrors = uniqWith(errors, e => e.reason && e.row);
```
Comment (outoftime) [inline src/validations/mergeValidations.js]: I don’t think this is doing quite what you intend—[`uniqWith`](https://lodash.com/docs/4.17.11#uniqWith) passes two arguments to the callback, which are two values to be compared. So, your function should take two parameters, and then return `true` if the `reason` and `row` properties have the same value across those two parameters.

Comment (Iula79) [reply src/validations/mergeValidations.js]: hi thanks for this I am testing now

**Pull request merged**